apps/docs/pages/guides/getting-started/quickstarts/kotlin.mdx

```diff
@@ -0,0 +1,322 @@
+  subtitle:
+    'Learn how to create a Supabase project, add some sample data to your database, and query the data from a Android Kotlin app.',
+  breadcrumb: 'Framework Quickstarts',
+}
+
+<StepHikeCompact>
+
+  <StepHikeCompact.Step step={1}>
+    <StepHikeCompact.Details title="Set up a Supabase project">
+
+    [Create a new project](https://app.supabase.com) in the Supabase Dashboard.
+
+    After your project is ready, create a table in your Supabase database using the [SQL Editor](https://app.supabase.com/project/_/sql) in the Dashboard. Use the following SQL statement to create all tables.
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+
+     ```sql SQL_EDITOR
+      -- Create products table
+      create table
+        public.products (
+          _id bigint generated by default as identity not null,
+          productid text not null,
+          name text null,
+          description text null,
+          price real null,
+          image text null,
+          category text null,
+          nutrition text null,
+          constraint products_pkey primary key (productid),
+        ) tablespace pg_default;
+
+      ````
+
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={2}>
+
+    <StepHikeCompact.Details title="Create an Android app with Android Studio">
+      Setup environment for Android Development following official document.
+      Open Android Studio > New > New Android Project
+
+    </StepHikeCompact.Details>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={3}>
+
+    <StepHikeCompact.Details title="Install the Supabase client library">
+    Setup Supabase libraries requires some others libraries from ktor, so make sure all dependencies are added.
+    Replace the dependency version placeholders `$supabase_version` and `$ktor_version` with the latest versions as shown below.
+    <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}> 
+      <a href="https://github.com/supabase-community/supabase-kt/releases" style={{ marginRight: '8px' }}> 
+          <img src="https://img.shields.io/github/release/supabase-community/supabase-kt?label=supabase-kt stable" alt="latest stable supabase-kt version"/> 
+      </a> 
+      <a href="https://github.com/ktorio/ktor/releases"> 
+          <img src="https://img.shields.io/github/v/release/ktorio/ktor.svg?label=ktor" alt="latest ktor version"/> 
+      </a> 
+    </div> 
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+      ```kotlin
+        implementation "io.github.jan-tennert.supabase:postgrest-kt-android:$supabase_version"
+        implementation "io.ktor:ktor-client-android:$ktor_version"
+        implementation "io.ktor:ktor-client-logging-jvm:$ktor_version"
+        implementation "io.ktor:ktor-client-core:$ktor_version"
+        implementation "io.ktor:ktor-client-cio:$ktor_version"
+        implementation "io.ktor:ktor-utils-jvm:$ktor_version"
```

Comment:
Is the "utils" and "logging" ktor artifact getting used? Also there are two Android engines used here, `CIO` and `Android`. Same with the Tutorial dependencies. `-android` and `-jvm` is also not needed.